packages/ui/dialogs/friend-detail/index.tsx

```diff
@@ -169,5 +158,5 @@ class RemoveFriend extends Component<Props, State> {
 
 export default connect((state) => ({ user: getUser(state)(), pendingTransactions: pendingTransactions(state), getTwoPartyBalance: getTwoPartyBalance(state),
   recentTransactions: recentTransactions(state), calculateBalance: calculateBalance(state), convertCurrency: convertCurrency(state), 
-  calculateUcacBalances: calculateUcacBalances(state), primaryCurrency: getPrimaryCurrency(state)() }),
+  primaryCurrency: getPrimaryCurrency(state)() }),
```

Comment:
This is odd usage of a Redux action. Actions do not have return values; instead they act upon the state and potentially produce side effects.

Instead, calculate and expose `primaryCurrency` in the state directly.